src/component/Text.tsx

```diff
@@ -190,6 +189,12 @@ const getWordsByLines = ({ width, scaleToFit, children, style, breakAll, maxLine
 
 const DEFAULT_FILL = '#808080';
 
+const reduceCSSCalc = (calc: string) => {
+  const tempEle = document.createElement('div');
+  tempEle.style.width = calc;
+  return tempEle.style.width.slice(5, -1).trim();
+};
```

Comment:
Adding this without any unit tests is not sufficient.

Reply:
Where will this behave different than `import reduceCSSCalc from 'reduce-css-calc';`
It will certainly not be identical, I assume - given the code size difference.

Reply:
> Adding this without any unit tests is not sufficient.

I'd be happy to add testing if it's needed. However, since it relies on browser functionality, a browser testing library like Cypress is required.

Reply:
> Where will this behave different than `import reduceCSSCalc from 'reduce-css-calc';` It will certainly not be identical, I assume - given the code size difference.

This code is referenced from https://drafts.csswg.org/css-values-3/#calc-syntax.
The differences I am aware of between reduce-css-calc and this code are in the calculation of unitless numbers and the return for invalid expressions.

Reply:
Right, so how would (a) unitless numbers and (b) invalid expressions change the product behaviour here?

Reply:
![image](https://github.com/recharts/recharts/assets/34399997/9d645681-448c-439e-9483-39616841203a)


I have made some modifications and also written a simple test code.
I excluded cases where the result of `reduce-css-calc` is `calc(~~)` because the dy attribute doesn't work correctly in those cases. 
Since invalid expressions are being ignored, i can expect the same behavior as before

Reply:
Wow, thanks for looking into performance. This alone is reason to look into this more.